src/components/Form/Form.tsx

```diff
@@ -13,23 +13,23 @@ const Form = ({ changeUserSentiment } : FormProps) : JSX.Element => {
       changeUserSentiment(Number(button.name));
   }
 
-  return (
-    <div className="form-container">
-      <section className="questionnaire">
+  return(
+    <section className="questionnaire">
         <h2>how are you feeling today?</h2>
-        <form className="sentiment-selection">
-          <button className="happy-btn" name="1" onClick={assignSentiment}></button>
-          <button className="neutral-btn" name="0" onClick={assignSentiment}></button>
-          <button className="sad-btn" name="-1" onClick={assignSentiment}></button>
-        </form>
-        <article className="wellnews-synopsis">
-          <p>WellNews uses your mood to curate a list of articles with your mental health in mind.</p>
-          <p>A negative mood will prioritize positive articles. <br/>A positive mood will result in a mix of positive, neutral, and negative news.</p>
-          <p>If you are unsure, select a neutral mood for a mix of articles that lean towards the positive.</p>
-          <p>You can change your mood preference at any time.</p>
-        </article>
-      </section>
-    </div>
+        <div className="sentiment-selection">
+          <div className="sentiments-container">
+            <button className="happy-btn" name="1" onClick={assignSentiment}></button>
```

Comment:
The accessibility audits are flagging the fact that our buttons are empty. The solution WAVE provides: "Place text content within the <button> element or give the <input> element a value attribute." Just making a note for us to discuss how to address this.

Reply:
Good catch. I think assigning a value of 'happy', etc. would be the best option to avoid interference with the image itself. If that sounds good enough, I can add 'happy', 'neutral', and 'sad' values during my refactor this afternoon.

Reply:
(GH deleted my response for some reason? Let me try again.)

Good catch - I think assigning values of 'happy', 'neutral', and 'sad' would be the best option since we don't want text on the buttons. I can add those during my refactor this afternoon if that sounds good to you!

Reply:
This is something that should be on my docket to update. I'm about to push my changes with router and will update this when I'm in there!

Reply:
Perfect! Thank you, Nadia. I'll just focus on the CSS changes for now to avoid merge conflicts. :)